Lib/pathlib.py

```diff
@@ -1166,6 +1189,210 @@ def walk(self, top_down=True, on_error=None, follow_symlinks=False):
 
             paths += [path._make_child_relpath(d) for d in reversed(dirnames)]
 
+    def absolute(self):
+        """Return an absolute version of this path by prepending the current
+        working directory. No normalization or symlink resolution is performed.
+
+        Use resolve() to get the canonical path to a file.
+        """
+        raise UnsupportedOperation(f"{type(self).__name__}.absolute()")
+
+    @classmethod
+    def cwd(cls):
+        """Return a new path pointing to the current working directory."""
+        return cls().absolute()
+
+    def expanduser(self):
+        """ Return a new path with expanded ~ and ~user constructs
+        (as returned by os.path.expanduser)
+        """
+        raise UnsupportedOperation(f"{type(self).__name__}.expanduser()")
+
+    @classmethod
+    def home(cls):
+        """Return a new path pointing to the user's home directory (as
+        returned by os.path.expanduser('~')).
```

Comment:
Perhaps a bit pedantic, but again on the point that we shouldn't assume contracts that would apply to a normal (Unix-biased) file system
```suggestion
        """Return a new path pointing to expanduser('~').
```